theme/templates/base.html

```diff
@@ -22,6 +22,8 @@
         {% endif %}
         <!-- Mobile nav script -->
         <script type="text/javascript" src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/scripts/mobile-nav.js"></script>
+        <!-- Google analytics script -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id={{ GOOGLE_ANALYTICS }}"></script>
```

Comment:
Move this after footer, together with the other script tag. And surround them with an if condition so that these tags are rendered only when th eGOOGLE_ANALYTICS setting is present.